websockets.md

```diff
@@ -87,7 +87,8 @@ A WebSocket client calls this method with the following query parameters to esta
       Once you establish a connection, you can keep it alive indefinitely.
       You remain authenticated for as long as you keep the connection open.
       You do not need to refresh the access token for an active connection
-      that lasts beyond the token's expiration time.
+      that lasts beyond the token's expiration time. In fact, a connection
```

Comment:
I'm waffling on "In fact." Does the sentence lose anything by losing that clause? No big; your call.

Reply:
I know it's gratuitous, but I really liked the "in fact" clause.

Reply:
Good by me.